modules/Rock.cmake

```diff
@@ -56,6 +56,9 @@ macro (rock_init PROJECT_NAME PROJECT_VERSION)
     rock_use_full_rpath("${CMAKE_INSTALL_PREFIX}/lib")
     include(CheckCXXCompilerFlag)
     include(FindPkgConfig)
+    
+    OPTION(ROCK_USE_CXX11 "Compile package using the C++11 standard" ON)
```

Comment:
There are external packages that in current versions rely on cpp11, like the widely used control/urdfdom version 1.0 from ROS.
Rather than having a long list in the overrides to enable it fo all packages using it, i'd rather have it enabled globally.

Btw.: Which are the supported OSes for rock that do not have c++11 features?